validphys2/src/validphys/arclength.py

```diff
@@ -134,7 +134,8 @@ def plot_arc_lengths(
             xvalues + shift,
             yvalues,
             yerr=(ylower, yupper),
-            fmt=".",
+            fmt='',
+            linestyle = '',
```

Comment:
Ok I am still not completely sure on whether I understood. However, the following modification should do it.

Reply:
See https://github.com/NNPDF/nnpdf/pull/1681/files#r1124281268.